bot.rb

```diff
@@ -31,8 +31,8 @@
         when "/help"
           send.(HELLO)
         when "/story"
-          current_step = 0
-          check = true
+          CURENT_STEP = 0
+          CHECK = true
```

Comment:
I hope, i made it well enough. Look just at bot_2.rb
bot.rb - is pacup plan for heroku